snap/local/epc.conf

```diff
@@ -0,0 +1,112 @@
+#####################################################################
+#                   srsEPC configuration file
+#####################################################################
+
+#####################################################################
+# MME configuration
+#
+# mme_code:         8-bit MME code identifies the MME within a group.
+# mme_group:        16-bit MME group identifier.
+# tac:              16-bit Tracking Area Code.
+# mcc:              Mobile Country Code
+# mnc:              Mobile Network Code
+# full_net_name     Display Name of the Network
+# short_net_name    Short Display Name of the Network
+# apn:		          Set Access Point Name (APN)
+# mme_bind_addr:    IP bind addr to listen for eNB S1-MME connnections
+# dns_addr:         DNS server address for the UEs
+# encryption_algo:  Preferred encryption algorithm for NAS layer 
+#                   (supported: EEA0 (default), EEA1, EEA2, EEA3)
+# integrity_algo:   Preferred integrity protection algorithm for NAS 
+#                   (supported: EIA0 (rejected by most UEs), EIA1 (default), EIA2, EIA3
+# paging_timer:     Value of paging timer in seconds (T3413)
+# request_imeisv:   Request UE's IMEI-SV in security mode command
+# lac:              16-bit Location Area Code.
+#
+#####################################################################
+[mme]
+mme_code = 0x1a
+mme_group = 0x0001
+tac = 0x0007
+mcc = 001
+mnc = 01
+mme_bind_addr = 127.0.1.100
+apn = srsapn
+dns_addr = 8.8.8.8
+encryption_algo = EEA0
+integrity_algo = EIA1
+paging_timer = 2
+request_imeisv = false
+lac = 0x0006
+
+#####################################################################
+# HSS configuration
+#
+# db_file:         Location of .csv file that stores UEs information.
+#
+#####################################################################
+[hss]
+db_file = user_db.csv
+
+#####################################################################
+# SP-GW configuration
+#
+# gtpu_bind_addr:   GTP-U bind address.
+# sgi_if_addr:      SGi TUN interface IP address.
+# sgi_if_name:      SGi TUN interface name.
+# max_paging_queue: Maximum packets in paging queue (per UE).
+#
+#####################################################################
+
+[spgw]
+gtpu_bind_addr   = 127.0.1.100
+sgi_if_addr      = 172.16.0.1
+sgi_if_name      = srs_spgw_sgi
+max_paging_queue = 100
+
+####################################################################
+# PCAP configuration
+#
+# Packets are captured to file in the compact format decoded by 
+# the Wireshark s1ap dissector and with DLT 150. 
+# To use the dissector, edit the preferences for DLT_USER to 
+# add an entry with DLT=150, Payload Protocol=s1ap.
+#
+# enable:   Enable or disable the PCAP.
+# filename: File name where to save the PCAP.
+#
+####################################################################
+[pcap]
+enable   = false
+filename = /tmp/epc.pcap
```

Comment:
Same thing here

Reply:
This is disabled by default as above, the difference that the user can enable it without the need to modify the filename config.
We could save those files to the data dir of the snap. like the logs.
Done.